airbyte-integrations/connectors/source-amazon-ads/Dockerfile

```diff
@@ -13,5 +13,5 @@ ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
 
-LABEL io.airbyte.version=3.1.2
+LABEL io.airbyte.version=3.1.3
```

Comment:
we should increment minor version if we're adding new fields to schemas, i.e. 3.2.0

Reply:
good point
updated